airbyte-integrations/connectors/source-shopify/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
+
+## 0.1.6
```

Comment:
DRY: let's not add a changelog file here since we add it in the public facing docs now